lms/templates/user_dropdown.html

```diff
@@ -0,0 +1,59 @@
+## mako
+<%page expression_filter="h"/>
+<%namespace name='static' file='static_content.html'/>
+
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
+%>
+
+% if uses_pattern_library:
```

Comment:
Nitpick: since it looks like the entire user dropdown is different depending on whether we're using the pattern library or not, it could make things a bit more ergonomic to have the pattern library / legacy versions refactored into separate files, and then this if/else could simply include/template in those partial templates.